can_tools/scrapers/official/federal/CDC/cdc_community_level.py

```diff
@@ -3,7 +3,7 @@
 from can_tools.scrapers.official.base import FederalDashboard
 
 
-COMMUNITY_LEVEL_MAP = {"Low": 0, "Medium": 1, "High": 2}
+COMMUNITY_LEVEL_MAP = {"Low": 0, "Medium": 1, "High": 2, "low": 0, "medium": 1, "high": 2}
```

Comment:
The lowercase versions are only used for a few places (Guam, America Samoa, etc.).  I wouldn't be surprised if it's a mistake and ends up getting fixed. 🤷